spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -2193,6 +2201,27 @@ def do_it
         module Quux
           extend(T::Sig)
           extend(T::Helpers)
+          extend(T::Generic)
+
+          A = type_template(:in)
+          B = type_template(:out)
+          C = type_template
+
+          D = type_member(fixed: Integer)
+          E = type_member(fixed: Integer, upper: Numeric)
```

Comment:
Can we add a test where the bound is itself generic?

Like here: https://sorbet.run/#%23%20typed%3A%20true%0A%0Aclass%20Foo%0A%20%20extend%20T%3A%3ASig%0A%20%20extend%20T%3A%3AGeneric%0A%0A%20%20E%20%3D%20type_member(upper%3A%20T%3A%3AArray%5BInteger%5D)%0A%0A%20%20sig%20%7Bparams(x%3A%20Integer).void%7D%0A%20%20def%20foo(x)%0A%20%20%20%20puts(x%20%2B%201)%0A%20%20end%0Aend

Reply:
Maybe also with a type modifier: https://sorbet.run/#%23%20typed%3A%20true%0A%0Aclass%20Foo%0A%20%20extend%20T%3A%3ASig%0A%20%20extend%20T%3A%3AGeneric%0A%0A%20%20E%20%3D%20type_member(upper%3A%20T.nilable(Integer))%0Aend

Reply:
Yeah, good call.